internal/app/api/v1/tests.go

```diff
@@ -436,7 +436,7 @@ func (s TestkubeAPI) UpdateTestHandler() fiber.Handler {
 		s.Log.Infow("updating test", "request", request)
 
 		var option *tests.Option
-		if request.Content != nil && (*request.Content) != nil && (*request.Content).Repository != nil && *(*request.Content).Repository != nil {
+		if request.Content != nil && (*request.Content) != nil && (*request.Content).Repository != nil {
```

Comment:
@nicufk why did you remove this check,  (*(*request.Content).Repository).Username can panic